site/content/in-dev/unreleased/getting-started/creating-a-catalog/s3/catalog-minio.md

```diff
@@ -17,9 +17,10 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-Title: Deploying Polaris on MinIO
+title: Creating a catalog on MinIO
+linkTitle: MinIO
 type: docs
-weight: 350
+weight: 200
 ---
 
 In this guide we walk through setting up a simple Polaris Server with local [MinIO](https://www.min.io/) storage.
```

Comment:
My thinking was to just give instructions for "manual" setup in this guide (copy from the docker compose file) and may be link to the docker compose example at the end of the guide (with minimal text, just indicating that the docker compose file shows and end-to-end integration example for MinIO). WDYT?

Reply:
Without refactoring this page, the whole Getting Started reorg is not quite complete, IMHO... Still, I'd be ok with merging these doc changes "as is". I can probably make a follow-up PR from my side. I guess it's easier than going through GH comments :sweat_smile: